index.html

```diff
@@ -8,89 +8,87 @@
     <link rel="stylesheet" href="styles/style.css" />
   </head>
   <body class="body">
-    <div class="top-section">
-      <div class="img-top">
-        <div class="header">
-          <div class="header-logo-left"></div>
-
-          <div class="header-logo-right">
-            <div class="stay-attraction">
-              <div class="stay">
-                Stays
-                <hr class="header-underline" />
-              </div>
-
-              <div class="attraction">Attractions</div>
+    <div class="section-container top-section">
+      <div class="header">
+        <div class="header-logo"></div>
+
+        <div class="header-nav">
+          <div class="header-section-container">
+            <div class="header-section-link">
+              Stays
+              <hr class="header-underline" />
             </div>
-            <div class="night"></div>
-            <div class="person"></div>
+
+            <div class="header-section-link">Attractions</div>
           </div>
+          <div class="night"></div>
+          <div class="person"></div>
         </div>
+      </div>
 
-        <div class="page">
-          <p class="page-text">
-            Discover stays <br />
-            to live, work or just relax
-          </p>
-        </div>
-        <form class="form" action="/" method="get">
-          <div class="label-input">
-            <label class="label-text" for="hotel"
-              >Your destination or hotel name</label
-            >
-            <div class="input-fon border-left-radius">
-              <input
-                class="input-1 border-left-radius"
-                id="hotel"
-                type="text"
-                name="hotel"
-                placeholder="New York"
-                required
-              />
-            </div>
+      <div class="header-section_title-container">
+        <h4 class="header-section_title_text">
+          Discover stays <br />
+          to live, work or just relax
+        </h4>
+      </div>
+      <form class="form" action="/" method="get">
+        <div class="label-input">
+          <label class="label-text" for="hotel"
+            >Your destination or hotel name</label
+          >
+          <div class="input-fon border-left-radius">
+            <input
```

Comment:
add bg-color for inputs
![image](https://user-images.githubusercontent.com/79144475/200139121-9530a3fd-cdb0-449e-aee6-7ec1d62cb644.png)

Reply:
Must be like
![image](https://user-images.githubusercontent.com/79144475/200139157-d96dd254-9463-457f-b4e0-a9c800b92abe.png)